torchdynamo/utils.py

```diff
@@ -80,6 +80,9 @@ def clear(self):
         self.values.clear()
         self.refs.clear()
 
+    def __delitem__(self, idx):
+        self._remove_id(id(idx))
```

Comment:
Update refs?

Reply:
I think `_remove_id` does:

```
def _remove_id(self, idx):
        if idx in self.refs:
            del self.values[idx]
            del self.refs[idx]
```
Or did you mean something else?

Reply:
You are correct